docs/azure_arc_jumpstart/azure_arc_app_svc/aks/aks_logic_app_arm_template/_index.md

```diff
@@ -0,0 +1,268 @@
+
+  * [**API Connection**](https://docs.microsoft.com/en-us/azure/connectors/apis-list)A connector provides prebuilt operations that you can use as steps in your Logic Apps workflows.
+
+  * [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) - Application Insights, a feature of Azure Monitor, is an extensible Application Performance Management (APM) service for developers and DevOps professionals. Use it to monitor your live applications.
+
+  * Azure Storage Account - The storage account deployed in this scenario is used for hosting the [queue storage](https://docs.microsoft.com/en-us/azure/storage/queues/storage-queues-introduction) and [blob storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction) where the Azure Logic App will be creating files in response to messages in a queue.
+
+  ![Additional Azure resources in the resource group](./26.png)
+
+* In this scenario, **a sample Jumpstart Azure Logic App** was deployed. To view the deployed Logic App, simply click the Azure Logic App resource.
+
+  ![Azure Logic App resource](./27.png)
+
+* You can view the Logic App workflow by clicking on "Workflows" and then clicking the workflow name "CreateBlobFromQueueMessage".
+
+  ![Azure Logic App detail](./28.png)
+
+  ![Azure Logic App detail](./29.png)
+
+* To demonstrate the messaging queuing element and to show how blobs are created when messages are read from the queue storage, the Azure Logic App deployment script also generates 10 sample queue messages. To view it, click on the newly created storage account and go to the "Queues" section where you will see the queue named "jumpstart-queue". Note that the queue will be empty because the workflow automatically deletes messages from the queue after creating a new blob in the "jumpstart-blobs" container.
+
+  ![Azure storage account](./30.png)
+
+  ![Azure storage queue](./31.png)
+
+* Go back to the Azure storage account and click on Containers. From here you will see the "jumpstart-blobs" container. Open this container and view the blobs that were created by the Logic App.
+
+  ![Azure storage container](./32.png)
+
+  ![Azure storage blobs](./33.png)
+
+* Alternatively, you can view the storage details using the Azure Storage Explorer client application installed automatically in the Client VM or using the Azure Storage Explorer portal-based view.
+
+  ![Azure Storage Explorer client application storage queue](./34.png)
+
+  ![Azure Storage Explorer portal-based view](./35.png)
+
+  ![Azure Storage Explorer portal-based view storage queue](./36.png)
+
+* To generate your own blobs using the Logic App, create a new message in the queue by using Azure Storage Explorer.
+
+  ![Add queue message](./37.png)
```

Comment:
I would duplicate _36.png_ here and add a red arrow pointing to the "Add Message" button

Reply:
good call, added